### ISSUE TITLE
Preprocessing with sklearn: a complete and comprehensive guide

### ISSUE THREAD

**Issue by RobertGarciadSilva** [khuyentran1401/machine-learning-articles#19]
## TL;DR
Useful guide to preprocessing data using sklearn.
### Article Link
https://towardsdatascience.com/preprocessing-with-sklearn-a-complete-and-comprehensive-guide-670cb98fcfb9
### Author
Steven Van Dorpe

## Key Takeaways

* Missing values
* Polynomial features
* Categorical features
* Numerical features
* Custom transformations
* Feature scaling
* Normalization

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*

**Comment by khuyentran1401** [comment]
Thank you for the helpful article!
